.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "semi": true,
+  "singleQuote": false,
+  "printWidth": 100,
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "useTabs": false
+}
```

Comment:
Please remove, we use `deno fmt`.

Reply:
Oh, I saw CONTRIBUTING.md. It need to be editted.